corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -998,7 +1003,15 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
                         .addTo(addressMap);
                     allCoordinates.push(homeCoordinates);
                 }
-                addressMap.fitBounds(allCoordinates, {maxZoom: 14});
+                if (allCoordinates.length > 0) {
+                    addressMap.fitBounds(allCoordinates, {maxZoom: 14});
+                } else if (defaultGeocoderLocation.bbox) {
+                    const bbox = defaultGeocoderLocation.bbox;
```

Comment:
I thought that is already code that zooms in on the projects location. What happens when you leave out `else if` but keep the if on line 1006?

Reply:
> this has the effect of using the project's default bbox specified bounds instead of a zoom level of 3

Yes, because the zoom wouldn't account for "size" of the project default. For example, "Hawaii" vs "USA"

> What happens when you leave out else if but keep the if on line 1006?

If `allCoordinates.length>0` is false, then it'll initialize the map and display it centered around latitude 30 and longitude 15 as defined [here](https://github.com/dimagi/commcare-hq/blob/b737d7c812d5e1304a1b36f8be1e8ca89bf148e0/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L896-L897)